### PR TITLE
New pull request branch

### PR DESCRIPTION
**Pull request by martinMr79**
Updated according to feedback from Oliver

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -102,9 +103,9 @@ <h3 class="h5">Kari Nordmann</h3>
         </div>
       </div>
     </section>
-    <section class="d-flex align-items-center justify-content-center">
-      <div class="card mb-2 row col-10">
-        <div class="border d-flex flex-row">
+    <section class="d-flex align-items-center justify-content-center row">
```
Comment (Fermain) [inline index.html]:
```suggestion
    <section class="align-items-center justify-content-center row">
```

Row is d-flex by default

File: index.html [hunk 2]
```diff
@@ -128,9 +129,9 @@ <h3 class="h5">Kari Nordmann</h3>
...
```
Comment (Fermain) [inline index.html]:
```suggestion
    <section class="align-items-center justify-content-center row">
```

**Review by Fermain — CHANGES_REQUESTED** [review]
Much better :) Just 2 small changes

**Comment by martinMr79** [comment]
Removed d-flex from line 79, 106 and 132

**Review by Fermain — APPROVED** [review]

**Pull request merged**